### DISCUSSION TITLE
Deployed github page, link takes me to a different website.

### DISCUSSION THREAD

**Discussion by GH0STP4WZ** [community/community#51576] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi, I have a bit of a weird situation, and I have absolutely no idea why this is happening.
I just deployed my own website via github pages. I was figuring out how to host a website Via github pages, so its bare. But, it does have enough code to make it distinct.

However, for whatever reason, following my site link, Riddim_GLiTCH.githhub.io/ is either taking me to a ww6 page, or is straight up telling me the the site doesn't exist. Which is strange. I am certain I did it correctly.  Whats going on here? Where is my website? I am so confused...

**Comment by airtower-luna** [comment, marked as answer]
> following my site link, Riddim_GLiTCH.githhub.io/ is either taking me to a ww6 page

That link is wrong:

1. GitHub Pages domains are under `github.io`, the link you're using has an extra `h`.
2. The domain name will use a normalized version of your username: All lower case, and `-` instead of the `_`.

The rest depends on your repository. Mind that if you want the repository to appear as a user site (under *just* the domain, without extra directories) it must be named exactly like your `github.io` domain.

**Reply by GH0STP4WZ** [reply]
forgive me for the link being wrong, my keyboard is not completely 100% anymore and in need of replacement.
According to your instructions, the correct link would be `riddim-glitch.github.io`. This returns an error 404 however.

**Reply by airtower-luna** [reply]
If you want your site to appear directly under `https://riddim-glitch.github.io/`, the repository must be named *exactly* `riddim-glitch.github.io` (that's the "user site" thing I mentioned above). With any other name the Pages site will be in a directory named just like the repository. For example, your [`Riddim_GLiTCH.githhub.io`](https://github.com/Riddim-GLiTCH/Riddim_GLiTCH.githhub.io) repo ends up deployed at https://riddim-glitch.github.io/Riddim_GLiTCH.githhub.io/.

**Reply by GH0STP4WZ** [reply]
I see. Thanks for the clarification! I'll be making these changes shortly.
